library/src/main/java/ch/decent/sdk/model/Operations.kt

```diff
@@ -281,3 +301,9 @@ class ContentSubmitOperation constructor(
     return "ContentSubmitOperation(size=$size, author=$author, coauthors=$coauthors, uri='$uri', quorum=$quorum, price=$price, hash='$hash', seeders=$seeders, keyParts=$keyParts, expiration=$expiration, publishingFee=$publishingFee, synopsis='$synopsis', custodyData=$custodyData, fee=$fee)"
   }
 }
+
+class SendMessageOperation constructor(
+    messagePayloadJson: String,
+    payer: ChainObject,
+    requiredAuths: List<ChainObject> = listOf(payer)
+) : CustomOperation(1, payer, requiredAuths, messagePayloadJson.toByteArray().hex())
```

Comment:
why is ID hardcoded to `1` here?

Reply:
custom operation subtype for message, should be documented in custom operation constructor

Reply:
fixed